std/math.d

```diff
@@ -4826,8 +4923,11 @@ version(Posix)
  * Returns the integer portion of x, dropping the fractional portion.
  *
  * This is also known as "chop" rounding.
+ *
+ * Returns:
+ *     A `real`.
  */
-real trunc(real x) @trusted nothrow @nogc
+auto trunc(real x) @trusted
```

Comment:
changed to `auto` to auto infer attributes like pure based on implementation without using template

Reply:
I feel the downside of not documenting the return type outweighs the benefit of attribute inference here.

Reply:
I added a note in the docs about the return type